e2e/pages/organizations/onc-acbs-atls/organizations.po.js

```diff
@@ -0,0 +1,88 @@
+const elements = {
+  organizationEditButton: '#edit-organization',
+  organizationName: '#organization-name',
+  organizationWebsite: '#organization-website',
+  saveOrganizationButton: '#chpl-organization-save',
+  organizationList: '.organizations-side-nav',
+  retireOrganization: '#organization-retired',
+  retirementDate: '#retirement-date',
+  manageUsersPanelHeader: '//*[contains(text(),"Manage Users")]',
+  errorMessage: '.text-danger.ng-scope',
+  newOrganizationGeneralInfo: '//*[@id="main-content"]/div/div/ui-view/chpl-onc-organizations/div[2]/div[2]/div[1]/div[2]',
```

Comment:
```suggestion
  newOrganizationGeneralInfo: '//chpl-onc-organizations/div[2]/div[2]/div[1]/div[2]',
```

You can probably start from the `chpl-onc-organizations` html element, as on these pages there should only ever be one of them, and anything to cut down on the number of nested elements in an xpath is good with me

Can do something similar with the next two lines, starting with `chpl-address` and `chpl-users` respectively